community-templates.json

```diff
@@ -14,6 +14,7 @@
         "branch": "templates-1.0.0",
         "path": "/templates/ts-apollo-postgres-backend"
       }
-    ]
+    ],
+    "external": true
```

Comment:
Idea is that this file will be for external ones so do we need flag

Reply:
We would need the flag somewhere. If not here then I could have it for the internal templates, something like `internal: true`?